### PR TITLE
English translation of README and zoom requirements documentation

### PR DESCRIPTION
**Pull request by ulvida**
Just text and human language in README...

### CODE REVIEW

File: README .es.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Zoom Recordings Downloader
+
+> Este [README en inglés](README.md)
+
+Una librería Python para descargar las grabaciones de cuentas Zoom.us
+
+## Arquitectura y requisitos
+
+Para acceder automáticamente a las grabaciones para descarga, se debe disponer de una cuenta Zoom con privilegios developer en la nube Zoom. Se podrá entonces descargar las grabaciones de todas las salas Zoom a las que la cuenta utilizada tenga acceso.
```
Comment (andrespias) [inline README .es.md]: I would change the beginning with this: "Para acceder automáticamente a la descarga de grabaciones, ..." or this: "Para poder descargar las grabaciones utilizando este utilitaro, ..."

File: README.md [hunk 2]
```diff
@@ -1,33 +1,41 @@
 # Zoom Recordings Downloader
 
+> This [README in Spanish](README.es.md)
+
 A Python library to download Zoom users recordings
 
-## Dependencias
+## Arquitecture and requirements
+
+To automatically access recordings for download, you must have a Zoom account with developer privilegies in Zoom platform. You can then dowonload the recordings of all the virtual rooms that your account has aqcces to.
```
Comment (andrespias) [inline README.md]: typo: "aqcces"

File: README.md [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
+pip3 install zoomus`
+pip3 install python-decouple
+pip3 install python-dateutil
 ```
 
-## Uso
+## Usage
 
-Primero deberá crear una JWT App accediendo con las credenciales de su cuenta en el [Marketplace de Zoom](https://marketplace.zoom.us/) para lo cual se recomiendoa seguir [esta documentación](https://marketplace.zoom.us/docs/guides/build/jwt-app).
+First you must create a JWT App, accessing with your credentials inti the [Zoom Marketplace](https://marketplace.zoom.us/), following this [this documentation](https://marketplace.zoom.us/docs/guides/build/jwt-app).
```
Comment (andrespias) [inline README.md]: typo: "inti"

File: README.md [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
 
-Luego de esto deberá crear un archivo `.env` dentro del mismo directorio donde ubique la librería `zoom_recordings.py`. Puede tomar como referencia el ejemplo [`example.env`](example.env). El `.env` es el archivo de configuración de la librería. Allí deberá indicar entre otras cosas:
-* las credenciales (`API_KEY` y `API_SECRET`) de su JWT App recientemente creada.
-* la fecha incial (`start_year`, `start_month`, `start_day_of_month`) a partir de la cual se quieren seleccionar las grabaciones (por su fecha de grabación).
-* rutas de directorios para ubicar las grabaciones descargadas (`download_root_path`) y los logs (`log_dir_path`).
+Then, a `.env` file must be created, in the same directory as the library `zoom_recordings.py`. Teh [`example.env`](example.env) file gives a reference for this tasc. This `.env` file is the configuration file of the library. It must particularly contain:
```
Comment (andrespias) [inline README.md]: typos: "Teh", "tasc".

File: README.md [hunk 5]
```diff
@@ -1,33 +1,41 @@
...
+* the credentials (`API_KEY` and `API_SECRET`) of the JWT App just created in the Zoom platform.
+* the initial date (`start_year`, `start_month`, `start_day_of_month`) since when the script shall download recordings (according to their creation date).
+* paths to save the downloaded recordings (`download_root_path`) and the logs (`log_dir_path`).
 
-Por último, para usar la librería `zoom_recordings.py`, puede crear otro archivo .py donde importar la funcion `download_recordings` y personalizar su invocación. Esta función recibe por parámetro la lista de usuarios (`users_selected`) para los cuales se desean descargar sus grabaciones. A continuación un ejemplo de su utilización:
+Finally, to run the library `zoom_recordings.py`, you can call it from a python program, creating another .py file to import the `download_recordings` and customize parameters. This function needs as parameters the list of users, or more precisely rooms (`users_selected`) which recordings you want to download. Heere is a use example:
```
Comment (andrespias) [inline README.md]: typo: "Heere"
Maybe also change "to import the `download_recordings`"  to something like "to import the `download_recordings` function" don't you think?

**Review by andrespias — COMMENTED** [review]
Reivew done!

File: README .es.md (hunk 1 above)
Comment (ulvida) [reply README .es.md]: Ok, that's better. And let's also change the wording in English.

File: README.md (hunk 3 above)
Comment (ulvida) [reply README.md]: "Int"` means "sun" in Quechua. The week end was cold...

**Comment by ulvida** [comment]
Thanks for the review, and sorry I didn't check twice.

**Comment by andrespias** [comment]
> Thanks for the review, and sorry I didn't check twice.

No problem. Thanks for your contribution!

**Pull request merged**